storybook/src/docs/components/AppIcons.tsx

```diff
@@ -3,7 +3,7 @@ import { Image } from '@amsterdam/design-system-react'
 export const AppleTouchIcon = () => (
   <div className="ams-docs-gallery">
     <figure className="ams-docs-figure">
-      <Image src="favicon/apple-touch-icon.png" width={180} height={180} />
+      <Image alt="apple-touch-icon.png (180px)" src="favicon/apple-touch-icon.png" width={180} height={180} />
```

Comment:
You shouldn't repeat visible text in the `alt` attribute, this creates noise for screen reader users.

```suggestion
      <Image alt="" src="favicon/apple-touch-icon.png" width={180} height={180} />
```

Same for the other cases